resources/views/login.blade.php

```diff
@@ -14,7 +14,11 @@
     @endif
     <style>
         body {
+            @if (config('voyager.multilingual.rtl'))
```

Comment:
This config doesn't exist in 1.x anymore.
Please refer to this line:
https://github.com/the-control-group/voyager/blob/58a45466660c99bf010e0431e0ab6d9d5392a6cb/resources/views/master.blade.php#L2